test-retroactive.sh

```diff
@@ -25,12 +25,12 @@ git add .gitattributes
 git checkout .
 git commit -am'Import big files into git-fat'
 
-git log --stat
+#git log --stat
 
 git fat find 10000 | awk '{print $1}' > fat-files
 git filter-branch --index-filter "git fat index-filter $(fullpath fat-files) --manage-gitattributes" --tag-name-filter cat -- --all
 
-git log --stat
+#git log --stat
```

Comment:
Is there a reason for commenting these lines?

Reply:
I missed it. Fixed